nodewatcher/modules/devices/tplink/wdr3600.py

```diff
@@ -0,0 +1,106 @@
+from nodewatcher.core.generator.cgm import base as cgm_base, protocols as cgm_protocols, devices as cgm_devices
+
+
+class TPLinkWDR3600v1(cgm_devices.DeviceBase):
+    """
+    TP-Link WDR3600v1 device descriptor.
+    """
+
+    identifier = 'tp-wdr3600v1'
+    name = "WDR3600 (v1)"
+    manufacturer = "TP-Link"
+    url = 'http://www.tp-link.com/'
+    architecture = 'ar71xx'
+    usb = True
+    radios = [
+        cgm_devices.IntegratedRadio('wifi0', "Integrated wireless radio (2.4 GHz)", [
+            cgm_protocols.IEEE80211BGN(
+                cgm_protocols.IEEE80211BGN.SHORT_GI_20,
+                cgm_protocols.IEEE80211BGN.SHORT_GI_40,
+                cgm_protocols.IEEE80211BGN.RX_STBC1,
+                cgm_protocols.IEEE80211BGN.DSSS_CCK_40,
+            )
+        ], [
+            cgm_devices.AntennaConnector('a1', "Antenna0")
+        ], [
+            cgm_devices.DeviceRadio.MultipleSSID,
+        ]),
+        cgm_devices.IntegratedRadio('wifi1', "Integrated wireless radio (5 GHz)", [
+            cgm_protocols.IEEE80211AN(
+                cgm_protocols.IEEE80211AN.SHORT_GI_20,
+                cgm_protocols.IEEE80211AN.SHORT_GI_40,
+                cgm_protocols.IEEE80211AN.RX_STBC1,
+                cgm_protocols.IEEE80211AN.DSSS_CCK_40,
+            )
+        ], [
+            cgm_devices.AntennaConnector('a2', "Antenna1")
+        ], [
+            cgm_devices.DeviceRadio.MultipleSSID,
+        ])
+    ]
+    switches = [
+        cgm_devices.Switch(
+            'sw0', "Switch0",
+            ports=6,
+            cpu_port=0,
+            cpu_tagged=True,
+            vlans=16,
+        )
+    ]
+    ports = [
+        cgm_devices.SwitchedEthernetPort(
+            'wan0', "Wan0",
+            switch='sw0',
+            vlan=2,
+            ports=[0, 1],
```

Comment:
yes, the switch config is the same like tp-link wdr4300.
https://wiki.openwrt.org/toh/tp-link/tl-wdr3600#switch_ports_for_vlans